src/wallet/wallet.h

```diff
@@ -666,7 +673,8 @@ class CWallet final : public WalletStorage, public interfaces::Chain::Notificati
     CAmount GetDebit(const CTransaction& tx, const isminefilter& filter) const;
     void chainStateFlushed(const CBlockLocator& loc) override;
 
-    DBErrors LoadWallet();
+    enum class do_init_used_flag { Init, Skip };
+    DBErrors LoadWallet(const do_init_used_flag do_init_used_flag_val = do_init_used_flag::Init);
```

Comment:
I haven't done any measurements with a large historical wallet, but is it a performance concern when we immediately initialize the address book on wallet creation?

Is it worth considering to lazy-initialize the address book on first use of `FindScriptPubKeyUsed`?

Reply:
I doubt it